pkg/agent/openflow/client.go

```diff
@@ -97,14 +96,13 @@ type Client interface {
 	// InstallEndpointFlows.
 	UninstallEndpointFlows(protocol binding.Protocol, endpoints []proxy.Endpoint) error
 
-	// InstallServiceFlows installs flows for accessing Service NodePort, LoadBalancer and ClusterIP. It installs the
-	// flow that uses the group/bucket to do service LB. If the affinityTimeout is not zero, it also installs the flow
-	// which has a learn action to maintain the LB decision. The group with the groupID must be installed before,
-	// otherwise the installation will fail.
-	// nodeLocalExternal represents if the externalTrafficPolicy is Local or not. This field is meaningful only when
-	// the svcType is NodePort or LoadBalancer.
-	// nested represents if the Service has the Endpoints which is other Service's ClusterIP.
-	InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16, nodeLocalExternal bool, svcType v1.ServiceType, nested bool) error
+	// InstallServiceFlows installs flows for accessing Service NodePort, LoadBalancer, ExternalIP and ClusterIP. It
+	// installs the flow that uses the group/bucket to do Service LB. If the affinityTimeout is not zero, it also
+	// installs the flow which has a learn action to maintain the LB decision. The group with the groupID must be
+	// installed before, otherwise the installation will fail.
+	// externalAddress indicates that whether the Service is externally accessible, like NodePort, LoadBalancer and ExternalIP.
+	// nested indicates that whether the Service is backed by Service IPs of other Services.
```

Comment:
I feel we should use "Endpoints of other Services", not "Service IPs", or you may say "nested, when setting to true, indicates the Service's Endpoints are ClusterIPs of other Services".

Reply:
"nested, when setting to true, indicates the Service's Endpoints are ClusterIPs of other Services" sounds good to me too. @hongliangl can you address it so we can proceed?

Reply:
Sure, wait a minute.

Reply:
Done